tools/clang/test/CodeGenSPIRV/method.byte-address-buffer.store.hlsl

```diff
@@ -2,6 +2,12 @@
 
 RWByteAddressBuffer outBuffer;
 
+struct S
+{
+    uint32_t x:8;
+    uint32_t y:8;
+};
```

Comment:
nit: shall we add a test checking we load the correct indices depending on the struct padding?
Ex:

```
struct S {
    uint32_t x : 8;
    uint64_t y : 8; // aligned on the 64bit boundary, hence 32bit uint padding inserted before.
};
```

Reply:
There are other tests that check that the layout of the spir-v type is correct. See op.struct.access.bitfield.sized.hlsl. I don't want to repeat all of those tests for every way that bitfields can interact with other features.